source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -125,6 +95,23 @@ method to retrieve the subsequent element of the cursor:
    :start-after: start manual cursor example
    :end-before: end manual cursor example
 
+.. _node-fundamentals-cursor-array:
+
+Return an Array of All Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For use cases that require all documents matched by a query to be held
+in memory at the same time, use `toArray() <{+api+}/classes/FindCursor.html#toArray>`__.
```

Comment:
Nit/Suggestion: The rest of the sections on this page follow the structure "...use the X method...":

```suggestion
in memory at the same time, use the `toArray() <{+api+}/classes/FindCursor.html#toArray>`__ method.
```